### PR TITLE
DEVDOCS-5012: [new] Webhooks, add metafield webhooks

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5012]
{Ticket number or summary of work}

## What changed?
Update callbacks article
* 

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

[DEVDOCS-5012]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-tgomez — DISMISSED** [review]
Looks good to my eyes.

File: models/webhooks/store_brand_metafield_updated.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/brand/metafield/updated:
+    description: This webhook is fired whenever a brand metafield is modified.
```
Comment (slsriehl) [inline models/webhooks/store_brand_metafield_updated.yml:4]:
```suggestion
    description: This webhook fires whenever a brand metafield is modified.
```

File: models/webhooks/store_cart_metafield_created.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/cart/metafield/created:
+    description: This webhook will fire whenever a new cart metafield is created.
```
Comment (slsriehl) [inline models/webhooks/store_cart_metafield_created.yml:4]:
```suggestion
    description: This webhook fires whenever a new cart metafield is created.
```

File: models/webhooks/store_brand_metafield_created.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/brand/metafield/created:
+    description: This webhook will fire whenever a new metafield on any object is created.
```
Comment (bc-traciporter) [inline models/webhooks/store_brand_metafield_created.yml:4]:
```suggestion
    description: This webhook fires whenever a new metafield on any object is created.
```

File: models/webhooks/store_brand_metafield_deleted.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/brand/metafield/deleted:
+    description: This webhook will fire whenever a brand metafield is deleted.
```
Comment (bc-traciporter) [inline models/webhooks/store_brand_metafield_deleted.yml:4]:
```suggestion
    description: This webhook fires whenever a brand metafield is deleted.
```

File: models/webhooks/store_cart_metafield_deleted.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/cart/metafield/deleted:
+    description: This webhook will fire whenever a cart metafield is deleted.
```
Comment (bc-traciporter) [inline models/webhooks/store_cart_metafield_deleted.yml:4]:
```suggestion
    description: This webhook fires whenever a cart metafield is deleted. 
```

File: models/webhooks/store_cart_metafield_updated.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  store/cart/metafield/updated:
+    description: This webhook is fired whenever a cart metafield is modified through the changes in its line items.
```
Comment (bc-traciporter) [inline models/webhooks/store_cart_metafield_updated.yml:4]:
```suggestion
    description: This webhook fires whenever a cart metafield is modified through the changes in its line items.
```

File: models/webhooks/store_inventory_location_metafield_created.yml [hunk 7]
```diff
@@ -0,0 +1,22 @@
+properties:
+  store/inventory/location/metafield/created:
+    description: This webhook will fire whenever a new inventory, location metafield is created.
```
Comment (bc-traciporter) [inline models/webhooks/store_inventory_location_metafield_created.yml:3]:
```suggestion
    description: This webhook fires whenever a new inventory, location metafield is created.
```

File: models/webhooks/store_inventory_location_metafield_deleted.yml [hunk 8]
```diff
@@ -0,0 +1,22 @@
+properties:
+  store/inventory/location/metafield/deleted:
+    description: This webhook will fire whenever a inventory, location metafield is deleted.
```
Comment (bc-traciporter) [inline models/webhooks/store_inventory_location_metafield_deleted.yml:3]:
```suggestion
    description: This webhook fires whenever a inventory, location metafield is deleted.
```

**Review by bc-traciporter — COMMENTED** [review]
removed "will"

**Review by slsriehl — APPROVED** [review]
excellent work

**Review by slsriehl — APPROVED** [review]
excellent work

**Pull request merged**